app/ts/context/TransferHistory.tsx

```diff
@@ -0,0 +1,36 @@
+import { Signal, useSignal } from '@preact/signals'
+import * as funtypes from 'funtypes'
+import { ComponentChildren, createContext } from 'preact'
+import { useContext } from 'preact/hooks'
+import { RECENT_TRANSFERS_CACHE_KEY } from '../library/constants.js'
+import { persistSignalEffect } from "../library/persistent-signal.js"
+import { createCacheParser, TransferSchema } from '../schema.js'
+
+export const TransferHistoryCacheSchema = funtypes.Union(
+	funtypes.Object({
+		data: funtypes.Array(TransferSchema),
+		version: funtypes.Literal('1.0.0')
+	})
+)
+
+export type TransferHistory = funtypes.Static<typeof TransferHistoryCacheSchema>
```

Comment:
this can also be called `TransferHistoryCacheSchema`. In interceptor we have often:

```ts
export type EthereumUnsignedTransaction = funtypes.Static<typeof EthereumUnsignedTransaction>
export const EthereumUnsignedTransaction = funtypes.Union(EthereumUnsignedTransactionLegacy, EthereumUnsignedTransaction2930, EthereumUnsignedTransaction1559)
```